Editor/Utility/SaveDuringPlay.cs

```diff
@@ -221,24 +221,21 @@ public bool ScanFields(string fullName, MonoBehaviour b)
         /// Recursively scan the MonoBehaviours of a GameObject and its children.
         /// For each leaf field found, call the OnFieldValue delegate.
         /// </summary>
-        public bool ScanFields(GameObject go, string prefix = null)
```

Comment:
I think you should revert this change and go back to your earlier implementation, with this difference: instead of checking whether the MonoBehaviour is a Cinemachine class, check if it has the SaveDuringPlay attribute.